mantidproject/mantid
# Migrate MBC Training to Sphinx

**Description of work.**

This PR moves the documentation for the basic training course across to Sphinx. This is part of an ongoing move to consolidate the documentation in one place.

**Report to:** [user name]/[nobody]. 
**To test:**



* Build the `docs-html` target of Mantid. I found that on OSX I had to build a regular target first then build the `docs-html` target
* Check the front index page of documentation 
* Follow the links to `Training/Mantid Basic Course`
* Check the content 
* Check that each of the pages has working figures and hyperlinks (including email link)
* Check that the text makes sense

Fixes #xxxx. 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
